benchmark/distributed_join.cu

```diff
@@ -108,6 +111,7 @@ void report_configuration()
   std::cout << "Communicator: " << COMMUNICATOR_NAME << std::endl;
   if (COMMUNICATOR_NAME == "UCX")
     std::cout << "Buffer communicator: " << USE_BUFFER_COMMUNICATOR << std::endl;
+  std::cout << "Compression: " << COMPRESSION << std::endl;
```

Comment:
I prefer flushing the buffer as early as possible. When the code encounters an error (e.g. segfault, CUDA error), I hope the information here is still printed out, instead of stuck in a buffer. Plus, it's not a performance critical part of the code.